tests/Mail/MailMailerTest.php

```diff
@@ -209,8 +210,11 @@ public function testGlobalToIsRespectedOnAllMessages()
         });
 
         $this->assertSame('taylor@laravel.com', $sentMessage->getEnvelope()->getRecipients()[0]->getAddress());
-        $this->assertStringNotContainsString('dries@laravel.com', $sentMessage->toString());
-        $this->assertStringNotContainsString('james@laravel.com', $sentMessage->toString());
+        $this->assertStringContainsString('X-To: nuno@laravel.com', $sentMessage->toString());
+        $this->assertStringContainsString('X-Cc: dries@laravel.com', $sentMessage->toString());
+        $this->assertStringContainsString('X-Bcc: james@laravel.com', $sentMessage->toString());
+        $this->assertFalse($recipients->contains('nuno@laravel.com'));
+        $this->assertFalse($recipients->contains('dries@laravel.com'));
```

Comment:
We should be a bit carefully here because we're not actually checking anymore if the CC: and BCC: headers are not present anymore (something the original tests implicitly did). I don't think BCC recipients, for example, are listed in the recipients of an envelope to begin with?